addons/mail/static/src/models/dropdown_menu_manager.js

```diff
@@ -0,0 +1,15 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { many } from '@mail/model/model_field';
+
+registerModel({
+    name: 'DropdownMenuManager',
+    fields: {
+        // FIXME: dependent on implementation that uses insert order in relations!!
+        items: many('DropdownMenuView', {
```

Comment:
`items` is a bit too generic as name, `dropdownMenuViews` would be the long version, clearly describing what kind of items (`DropdownMenuView`) we have.

Current name is especially confusing because `items` is also used to reference `DropdownItemView` in another context related to this feature.

Reply:
The name of field is contextual to model, so it's very clear that `DropdownMenuManager/items` is not the same as `DropdownMenuView/items`.